### PR TITLE
Add json6902 patch filter based on kyaml libraries

### PR DESCRIPTION
**Pull request by mortent**
@pwittrock

### CODE REVIEW

**Comment by pwittrock** [comment]
@mortent Run the `travis/check-go-mod.sh` script

**Comment by mortent** [comment]
@pwittrock fixed

File: plugin/builtin/patchjson6902transformer/PatchJson6902Transformer.go [hunk 1]
```diff
@@ -87,14 +91,20 @@ func (p *plugin) Transform(m resmap.ResMap) error {
...
+		if err != nil {
+			return err
+		}
+		modifiedObj, err := p.decodedPatch.Apply(rawObj)
+		if err != nil {
+			return errors.Wrapf(
+				err, "failed to apply json patch '%s'", p.JsonOp)
+		}
+		return obj.UnmarshalJSON(modifiedObj)
+	} else {
```
Comment (monopole) [inline plugin/builtin/patchjson6902transformer/PatchJson6902Transformer.go:105]: if p.yamlsupport {
  return 
}
// keep going.

Comment (monopole) [reply plugin/builtin/patchjson6902transformer/PatchJson6902Transformer.go:105]: /lgtm

(just a nit)

**Pull request merged**